fedimint-api-client/src/api/mod.rs

```diff
@@ -622,11 +623,35 @@ where
 pub struct WebsocketConnector {
     peers: BTreeMap<PeerId, SafeUrl>,
     api_secret: Option<String>,
+
+    /// List of overrides to use when attempting to connect to given
+    /// `PeerId`
+    ///
+    /// This is useful for testing, or forcing non-default network
+    /// connectivity.
+    pub connection_overrides: BTreeMap<PeerId, SafeUrl>,
```

Comment:
I'll touch the API announcements soon anyway, but it might be worth disambiguating between the different sources of API URLs here